### PR TITLE
[RFR] Remove pm2 from tests

### PR DESCRIPTION
**Pull request by alexisjanvier**

### CODE REVIEW

**Comment by Kmaschta** [comment]
I'm clearly not in favor of `@` in the makefile. Theyl were removed with #135

File: src/frontend/e2e/products.js [hunk 1]
```diff
@@ -4,11 +4,10 @@ import expect from 'expect';
 import driver from '../../common/e2e/lib/chromeDriver';
 
 describe('Products', function () {
-    this.timeout(15000);
+    this.timeout(150000);
```
Comment (alexisjanvier) [inline src/frontend/e2e/products.js]: debug spotted

**Comment by alexisjanvier** [comment]
@Kmaschta I added `@`  only for `echo` ... I'm not sure this `echo` are realy useful for debugging?

**Comment by Kmaschta** [comment]
Nope, you're right. I've just saw the travis build!

File: package.json [hunk 2]
```diff
@@ -102,6 +102,7 @@
     "html-webpack-plugin": "2.24.0",
     "http-server": "0.9.0",
     "json-loader": "0.5.4",
+    "koa-static": "next",
```
Comment (djhi) [inline package.json]: use the `3.0.0` release instead of next

**Review by djhi — CHANGES_REQUESTED** [review]
This is neat :+1:

**Review by djhi — APPROVED** [review]

**Pull request merged**